### DISCUSSION TITLE
Error get data

### DISCUSSION THREAD

**Discussion by syeillah** [Budibase/budibase#4665] [Help]
Hi, just wondering when i try to open my page (data source: external) but suddently some errors happen with these notif, may i know why?

<img width="683" alt="Screen Shot 2022-02-23 at 21 11 11" src="https://user-images.githubusercontent.com/99949174/155336213-65a9ad29-8caa-4786-9f4f-d42c30a8dafd.png">

**Comment by mjashanks** [comment]
Hey @syeillah - could you 
1. go to your network tab (in dev tools) 
2. find the query that is failing `/queries/query_datasource....` 
3. Click on it
4. Goto the `Response` tab
5. Past the result of that in here. That will give me more information.

I'll take a guess though - are you sure that the logged in user has access to the query?
